clouddriver-dcos/src/main/groovy/com/netflix/spinnaker/clouddriver/dcos/deploy/ops/job/RunDcosJobAtomicOperation.groovy

```diff
@@ -68,10 +68,15 @@ class RunDcosJobAtomicOperation implements AtomicOperation<DeploymentResult> {
 
     task.updateStatus BASE_PHASE, "Job ${jobName} has been started."
 
-    // TODO We will want to change location to use regions like apps once that is supported.
+    // We are kinda hacking our own name together here since applications cannot be "batch" jobs in DC/OS land currently.
+    // Stack = Metronome job name
+    // Detail = Job mesos task id
+    def friggaJobName = "${description.application}-${jobRun.jobId}-${jobRun.id}-v000".toString()
```

Comment:
Right now it doesn't quite feel worth it since it has limited use. I guess I could still go ahead and create it though.